packages/core/src/LottieView/index.web.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { forwardRef, useImperativeHandle, useRef, useCallback } from 'react';
+import type { LottieViewProps } from '../types';
+import { parsePossibleSources } from './utils';
+
+let DotLottiePlayer, Player;
+try {
+    DotLottiePlayer = require('@dotlottie/react-player').DotLottiePlayer;
+    Player = require('@lottiefiles/react-lottie-player').Player;
+} catch (e) { }
+
+const LottieView = forwardRef(
+  (
+    {
+      source,
+      speed,
+      loop,
+      webStyle,
+      autoPlay,
+      hover,
+      direction,
+      progress,
+      onAnimationFailure,
+      onAnimationFinish,
+      onAnimationLoop,
+    }: LottieViewProps,
+    ref: any,
+  ) => {
+    const sources = parsePossibleSources(source);
+    const isLottie =
+      sources.sourceName?.includes('.lottie') || !!sources.sourceDotLottieURI;
+    const lottieSource = sources.sourceDotLottieURI || sources.sourceName;
```

Comment:
Would you please try loading json file from remote url? 

Try this URL: https://raw.githubusercontent.com/lottie-react-native/lottie-react-native/master/apps/paper/src/animations/Watermelon.json

Tried disabling CORS but not luck.

Reply:
Fixed!